tests/out/glsl/skybox.vs_main.Vertex.glsl

```diff
@@ -21,15 +21,15 @@ void main() {
     int tmp2_ = 0;
     tmp1_ = (int(vertex_index) / 2);
     tmp2_ = (int(vertex_index) & 1);
-    int _expr10 = tmp1_;
-    int _expr16 = tmp2_;
-    vec4 pos = vec4(((float(_expr10) * 4.0) - 1.0), ((float(_expr16) * 4.0) - 1.0), 0.0, 1.0);
-    vec4 _expr27 = _group_0_binding_0.view[0];
```

Comment:
To review these changes, I reverted the change to the GLSL back end that used the `BAKE_PREFIX`, and verified that the only remaining changes to the GLSL output were benign. It would be nice to just leave such changes in separate PRs, since they're mechanical, and it creates a lot of chaff to sort through looking for interesting changes.

Reply:
ok, that's fair